mcc_generated_files/pin_manager.h

```diff
@@ -59,6 +59,78 @@
 #define PULL_UP_ENABLED      1
 #define PULL_UP_DISABLED     0
 
+// get/set channel_VBAT aliases

```

Comment:
These are new. Where are they coming from? They should not be needed.

Reply:
These where generated by the MCC, I can try to delete them if needed.